lib/hydra/works/services/generic_file/persist_derivative.rb

```diff
@@ -9,17 +9,29 @@ class PersistDerivative < Hydra::Derivatives::PersistOutputFileService
     # The purpose of this Service is for use as an alternative to the default Hydra::Derivatives::PersistOutputFileService.  It's necessary because the default behavior in Hydra::Derivatives assumes that you're using LDP Basic Containment. Hydra::Works::GenericFiles use IndirectContainment.  This Service handles that case.
     # 
     # @param [Hydra::Works::GenericFile::Base] object the file will be added to
-    # @param [#read] file the dervivative filestream optionally responds to :mime_type
-    # @option opts [RDF::URI or String] type URI for the RDF.type that identifies the file's role within the generic_file
+    # @param [#read or String] file the derivative filestream optionally responds to :mime_type
+    # @param [String] extract file type symbol (e.g. :thumbnail) from Hydra::Derivatives created destination_name
     # @option opts [Boolean] update_existing when set to false, always adds a new file.  When set to true, performs a create_or_update
     # @option opts [Boolean] versioning whether to create new version entries
 
-    def self.call(object, file, opts={})
-      type = opts.fetch(:type, ::RDF::URI("http://pcdm.org/use#ServiceFile"))
+    def self.call(object, file, destination_name, opts={})
+      type = destination_name.gsub(/^original_file_/, '').to_sym
       update_existing = opts.fetch(:update_existing, true)
       versioning = opts.fetch(:versioning, true)
-      Hydra::Works::AddFileToGenericFile.call(object, file, type, update_existing: update_existing, versioning: versioning)
+
+      # copy original_file stream to audio derivative if original_file is .ogg or .mp3
+      file = object.original_file.content if self.audio_format_match?(object.original_file.mime_type, type)
```

Comment:
Yes this behavior was expected in Sufia according to the [mp3 spec](https://github.com/projecthydra-labs/hydra-works/pull/183/files#diff-8b0e62a1ec782e7f1facccb31b5641caR129) and [ogg spec](https://github.com/projecthydra-labs/hydra-works/pull/183/files#diff-8b0e62a1ec782e7f1facccb31b5641caR143). If the original audio is already in the format of one of its derivatives, this would copy over the original file's bit stream to attach to the object. I'm unsure of what the benefits actually are of doing this. Especially considering the way I implemented that behavior, an mp3/ogg derivative would be created (not saving any time), and we have a duplicate file stream (not saving any space).

Since the audio derivatives are going away from Works, I'll open an issue/question in sufia-core so we can address this.